doc/modules/model_evaluation.rst

```diff
@@ -109,18 +110,18 @@ their callable needs to be passed to :func:`make_scorer` together with the
 value of the user-settable parameters. See :ref:`scoring` for more
 details on :func:`make_scorer` usage.
 
-==============================================     ==================================
-Function                                           User-settable parameter name
-==============================================     ==================================
+=====================================  =========  ==============================================
+Function                               Parameter  Example usage
+=====================================  =========  ==============================================
 **Classification**
-:func:`metrics.fbeta_score`                        ``beta``
+:func:`metrics.fbeta_score`            ``beta``   ``make_scorer(fbeta_score, beta=2)``
 
 **Regression**
-:func:`metrics.mean_tweedie_deviance`              ``power``
-:func:`metrics.mean_pinball_loss`                  ``alpha``
-:func:`metrics.d2_tweedie_score`                   ``power``
-:func:`metrics.d2_pinball_score`                   ``alpha``
-==============================================     ==================================
+:func:`metrics.mean_tweedie_deviance`  ``power``  ``make_scorer(mean_tweedie_deviance, power=1)``
+:func:`metrics.mean_pinball_loss`      ``alpha``  ``make_scorer(mean_pinball_loss, alpha=0.95)``
+:func:`metrics.d2_tweedie_score`       ``power``  ``make_scorer(d2_tweedie_score, power=1)``
+:func:`metrics.d2_pinball_score`       ``alpha``  ``make_scorer(d2_pinball_score, alpha=0.95)``
+=====================================  =========  ===============================================
 
 
 Usage examples:
```

Comment:
One last thing before to merge: I think that we should move the example below above the table that you added because it does not rely on `make_scorer`. However, we could add a similar example to show that we can pass a scorer object obtained via `make_scorer`.

In this case, we have a usage example for each configuration.

Reply:
We had some back and forth already about the exact position of the table. I feel if it is below the example, and just above "Defining your scoring strategy from metric functions" section, then it might as well be within that section. Perhaps @ogrisel can chime in too? 

And if I'm understanding correctly what you mean by "similar example to show that we can pass a scorer object obtained via make_scorer", then that example is in the 3.3.1.2. section already too?

Reply:
So the example right now is:

```
Usage examples:

    >>> from sklearn import svm, datasets
    >>> from sklearn.model_selection import cross_val_score
    >>> X, y = datasets.load_iris(return_X_y=True)
    >>> clf = svm.SVC(random_state=0)
    >>> cross_val_score(clf, X, y, cv=5, scoring='recall_macro')
    array([0.96..., 0.96..., 0.96..., 0.93..., 1.        ])
```

I feel strange that it comes right after the table that mentioning that you should `make_scorer`

Would it not be more friendly to have:


```python
====
Table showing string metric
====


    >>> from sklearn import svm, datasets
    >>> from sklearn.model_selection import cross_val_score
    >>> X, y = datasets.load_iris(return_X_y=True)
    >>> clf = svm.SVC(random_state=0)
    >>> cross_val_score(clf, X, y, cv=5, scoring='recall_macro')
    array([0.96..., 0.96..., 0.96..., 0.93..., 1.        ])

Narration about `make_scorer` only metric

===
Table for `make_scorer` metric
===

Usage example:

    >>> from sklearn import svm, datasets
    >>> from sklearn.metrics import fbeta_score, make_scorer
    >>> from sklearn.model_selection import cross_val_score
    >>> X, y = datasets.load_iris(return_X_y=True)
    >>> clf = svm.SVC(random_state=0)
    >>> scorer = make_scorer(fbeta_score, beta=2)
    >>> cross_val_score(clf, X, y, cv=5, scoring=scorer)
    array([...])
```

It might be slightly redundant with the section below but this is just a short example usage.

But indeed @ogrisel had maybe another opinion, so let's see what I think?

Reply:
Agree that this would be a more logical way to structure it, and yes, also agree on the redundancy since the next section talks about make_scorer in more detail and gives that exact same example.

Reply:
Alright, let's move back the second table at the beginning of the "3.3.1.2. Defining your scoring strategy from metric functions" section. We can start the section with a short sentence such as the one you wrote:

> The following metrics functions are not implemented as named scorers. They cannot be passed to the scoring parameters; instead their callable needs to be passed to make_scorer together with the value of the user-settable parameters

(without link to the section since we are already in it).

Then you can simplify the paragraph that starts with the "Many metrics are not given names to ..." to remove redundancies but keep introducing the usage example with the snippet that shows how to combine make_scorer with user settable parameters for a grid search.

Then, after the example snippet, finally insert the paragraph with the two bullet points:

![image](https://github.com/scikit-learn/scikit-learn/assets/89061/3b777a8e-7e2a-4e76-ab6a-2a3972d0ddcf)

This is not as important as the rest so I would move it to the end of the section but it's still useful information to speak about the naming conventions of the metric functions and how to set the `higher_is_better` keyword parameter in a consistent way.